packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -46,11 +47,40 @@ class Scrollbar extends StatefulWidget {
   /// Typically a [ListView] or [CustomScrollView].
   final Widget child;
 
+  /// The [ScrollController] used to implement Scrollbar dragging.
+  ///
+  /// Scrollbar dragging is started with a long press or a drag in from the side
+  /// on top of the scrollbar thumb, which enlarges the thumb and makes it
+  /// interactive. Dragging it then causes the view to scroll. This feature was
+  /// introduced in iOS 13.
+  ///
+  /// The scrollbar uses the [PrimaryScrollController] of the context by default.
+  /// If you are using a custom [ScrollController] for
+  ///
+  /// Here is an example of using a custom [ScrollController]:
+  ///
+  /// {@tool sample}
+  ///
+  /// ```dart
+  /// final ScrollController _controller = ScrollController();
+  ///
+  /// build(BuildContext context) {
+  ///   return Scrollbar(
+  ///     controller: _controller,
+  ///     child: ListView.builder(
+  ///       itemCount: 150,
+  ///       itemBuilder: (BuildContext context, int index) => Text('item $index'),
+  ///     ),
+  ///   );
+  /// }
+  /// ```
+  /// {@end-tool}
```

Comment:
If this is a verbatim copy of the docs for controller in CupertinoScrollbar, it should be referenced using `@macro` and `@template`.  See obscureText in [EditableText](https://github.com/flutter/flutter/blob/8c6d60bcfd71db2cd98d337d8466f9142919b7a2/packages/flutter/lib/src/widgets/editable_text.dart#L447) and [TextField](https://github.com/flutter/flutter/blob/8c6d60bcfd71db2cd98d337d8466f9142919b7a2/packages/flutter/lib/src/material/text_field.dart#L450), for example.